docs/messaging/messaging-spans.md

```diff
@@ -216,6 +216,7 @@ Span kind SHOULD be set according to the following table, based on the operation
 | `send`         | `PRODUCER` if the context of the "Send" span is used as creation context, otherwise `CLIENT`. |
 | `receive`      | `CLIENT`    |
 | `process`      | `CONSUMER`  |
+| `settle`       | `CLIENT` if the span models a synchronous call to the intermediary. Otherwise `INTERNAL` |
```

Comment:
we use SHOULD right above this table

 https://github.com/open-telemetry/semantic-conventions/blob/d5d2b9d2a27f5983a5930699e0d40d5aba5c06fe/docs/messaging/messaging-spans.md?plain=1#L211

So I wonder if we really need to explicitly call out the `INTERNAL` case.

Also the "to the intermediary" is not always the case. E.g. Azure EventHubs settles using some external service (such as Azure Blob Storage, Redis or something else). 

So maybe just

```suggestion
| `settle`       | `CLIENT` |
```

?

Reply:
Hum that makes sense. I guess just having client is really fine.